### DISCUSSION TITLE
SOZ_API: Table not existing

### DISCUSSION THREAD

**Discussion by Chamallaw** [SOZ-Faut-etre-Sub/SOZ-FiveM-Server#251] [Q&A] [answered]
Bonjour,

J'utilise actuellement la base en local sur ordinateur windows.
J'ai fait en sorte qu'on puisse jouer à plusieurs dessus ainsi que dev à plusieurs.

Le soucis qu'il me reste c'est concernant l'API, au Login des users il y à des erreurs dans la console.

La première est le fait qu'un query est exécuté sur la table soz_api, sauf que cette dernière n'existe pas dans la BDD locale après la migration SQL Est-ce normal ? Si non, comment l'ajouter ?

La seconde est la récupération du JWT Token car en timeout, je suppose que ç à également voir avec l'API ?

Merci par avant pour vos retours,
Chama, l'apprenti développeur :D

**Comment by norfair00** [comment]
Perso quand quelqu'un d'autre que moi join y'a tout qui crash
J'ai une fresh install

**Reply by Chamallaw** [reply]
Change ta version de build FiveM, j'avais le même soucis, je t edonne le numéro de build que j'utilise ce soir si j'y penses

**Reply by norfair00** [reply]
Bah j'ai celui qui donne dans les config

**Reply by Chamallaw** [reply]
Ouais j'avais également laissé celle de base, et le premier connecté ça marchait, mais le deuxième faisait que crash au chargement. Et c'est toujours le premier connecté qui ne crashait pas.

J'utilise le build version 2802 si jamais tu veux test avec ça, suffit de changer dans la config et restart

**Comment by Laikker** [comment, marked as answer]
soz_api est créé et populé par la partie web qui n'est pas OpenSource.
De même le JWT est utilisé pour se connecter au site web de depuis le jeu.
Comme ce n'est pas OpenSource, à toi de gérer de ton coté comment tu veux modifier ça pour ton serveur

**Reply by Chamallaw** [reply]
Ok et cette API elle permet de faire quoi exactement ? 
C'est pour gérer les whitelist et la connexion au site depuis le jeu ?

Si c'est le cas, Il existe un moyen de la désactiver ou faut-il supprimer les appels à cette dernière ?

**Reply by Laikker** [reply]
C'est pour gérer les whitelist et la connexion au site depuis le jeu ?
 - oui
 
Il n'y a pas de moyen de le désac, il faut que tu changes le code chez toi

**Reply by Chamallaw** [reply]
Merci pour tes réponses et ton temps :)
